update_existing_families.py

```diff
@@ -130,17 +164,20 @@ def save_as_csv_file(filename, header_fields, data):
             studentrecord['SCHOOL'] = 'ALUM'
         elif dp_record['SCHOOL'] != 'ALUM':
             studentrecord['SCHOOL'] = 'NOBSD'
-        if dp_record != studentrecord:
-            dp_import_studentrecords.append(studentrecord)
+        if dp_record_minimal != studentrecord:
```

Comment:
@kdawgsf, do you mind adding the following in comments in the code: 
`#The two records will be the same if:`
`# (a) we run this mid-year, OR`
`# (b) a student is already NOBSD`
`#The same process works whether we're running a summer update or a mid-year update.`